.buildkite/build_resources.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+import os
+import sys
+import yaml
+import http.client
+import json
+
+pipeline = {'steps': [
+    {'label': 'trigger-build-resources',
+     'trigger': 'vmm-reference-build-resources',
+     'skip': 'No changes in resources or not a Pull Request'}
+]}
+
+pull_request = os.environ.get('BUILDKITE_PULL_REQUEST')
+if pull_request and pull_request.isnumeric():
```

Comment:
Can we add a comment here saying that we do the `isnumeric()` check because `pull_request` will be "false" if it's not a pull request or something like that?